content/en/docs/home/_index.md

```diff
@@ -27,6 +27,11 @@ cards:
     description: "Look up common tasks and how to perform them."
     button: "View User Documentation"
     button_path: "/docs/user-documentation"
+  - name: networking
```

Comment:
Well, IMHO we shouldn't keep a 1:1 mapping between the index on the left, and the cards:
<img width="1505" alt="image" src="https://user-images.githubusercontent.com/1176434/157210060-cd3df58b-d8c7-4161-ad0c-8ddc8d75a9d4.png">

Otherwise what's the value of having two visualizations if they are exactly the same?

Reply:
It simply helps highlight other parts which could be important.

Reply:
So "Networking" is a child page of "User Docs", but we're just presenting it in the front page as a "card", is that correct?